exercises/bank-account/canonical-data.json

```
@@ -0,0 +1,317 @@
+{
+  "exercise": "bank-account",
+  "comments": [
+    "This exercise is a good candidate to practice concurrency, which",
+    "may not be available or possible in the implementing language track.",
+    "In this case, all tests can be executed on a single thread. If there",
+    "are tests that don't work using a single thread, these should be",
+    "marked with a concurrency scenario.",
+    "----------------------------------------------------------------",
+    "For tests which requires multiple expected values/errors",
+    "Should put the expected values/errors as a field of each operation",
+    "And then the last expected value/error should be the overall expected value for the case",
```

I would like to suggest that the overall `expected` value for the case be mentioned first, since it is more important.
It's more important because it's always applicable, regardless of whether any operations also include an `expected` value. Therefore:

```suggestion
    "The overall expected value for the case should be the result of the last operation.",
    "If any operation other than the last one has an expected result,",
    "the expected value/error should be a field of that operation",
```